playbooks/roles/scorm/tasks/scorm_players.yml

```diff
@@ -0,0 +1,32 @@
+
+- name: create missing scormplayers dir
+  file: >
+    path="{{ scorm_player_dir.path }}"
+    state="directory"
+    owner="{{ scorm_player_dir.owner }}"
+    group="{{ scorm_player_dir.group }}"
+    mode="{{ scorm_player_dir.mode }}"
+
+- name: install read-only ssh key for the scormplayer repo(s)
+  copy: >
+    content="{{ EDXAPP_AUTH_EXTRA.SCORMPLAYER_GIT_IDENTITIES[item.key] }}" dest=/tmp/scormplayer_{{ item.key }}_identity
+    force=yes owner={{ scorm_player_dir.owner }} mode=0600
+  with_dict: "{{scorm_config.SCORM_PLAYER_BACKENDS|default({})}}"
+
+- name: checkout player repo
+  git: >
+    dest={{ scorm_player_dir.path }}/{{item.key}} repo={{ item.value.repo }} version="{{ item.value.version }}"
+    accept_hostkey=yes key_file=/tmp/scormplayer_{{ item.key }}_identity
+    ssh_opts="-o StrictHostKeyChecking=no"
+  environment:
+    GIT_SSH: "{{ edxapp_git_ssh }}"
```

Comment:
We have both `edxapp_git_ssh` and `key_file` in the same command. I think `edxapp_git_ssh` is useless here.

Reply:
True, thanks.  I removed it.